tools/genomespace/genomespace_exporter.xml

```diff
@@ -1,54 +1,42 @@
-                #end if
-            #end for
-        #else:
-            --subdirectory "${ binascii.hexlify( 'galaxy_export' ) }"
-            --subdirectory "${ binascii.hexlify( str( $base_url ).split( '://', 1 )[-1] ) }" ##Protocol removed by request
-        #end if
+        --token "${ binascii.hexlify(str(token).encode('utf-8')) }"
+        --input_file "${ binascii.hexlify( str($input1).encode('utf8') ) }"
         #if $filename:
-            --filename "${ binascii.hexlify( str( $filename ) ) }"
+        --target_url "${ binascii.hexlify( str($target_folder + '/' + str( $filename )).encode('utf8') ) }"
         #else:
-            --filename "${ binascii.hexlify( "Galaxy History Item %s (%s) - %s: %s.%s" % ( $__app__.security.encode_id( $input1.id ), $__app__.security.encode_id( $output_log.id ), $input1.hid, $input1.name, $input1.ext ) ) }"
+        --target_url "${ binascii.hexlify( ($target_folder + '/' + 'Galaxy History Item %s (%s) - %s: %s.%s' % ( $__app__.security.encode_id( $input1.id ), $__app__.security.encode_id( $output_log.id ), $input1.hid, $input1.name.replace('/', '_'), $input1.ext )).encode('utf8') )  }"
         #end if
-        --file_type "${input1.ext}"
-        --content_type "${input1.get_mime()}"
-        --log "${output_log}"
-        --genomespace_toolname="\${GENOMESPACE_TOOLNAME:-Galaxy}"
     </command>
     <inputs>
         <param format="data" name="input1" type="data" label="Send this dataset to GenomeSpace" />
         <param name="base_url" type="baseurl" />
-        <param name="subdirectory" type="drill_down" display="radio" hierarchy="exact" multiple="False" optional="True" label="Choose Target Directory" dynamic_options="galaxy_code_get_genomespace_folders( genomespace_site = 'prod', trans=__trans__, value=__value__, input_dataset=input1, base_url=base_url )" help="Leave blank to generate automatically"/>
-        <param name="filename" type="text" size="80" label="Filename" help="Leave blank to generate automatically" />
+        <!-- If using this tool through bioblend, the genomespace_browser parameter should contain the path to the GenomeSpaceFile + the security token
+             separated by a ^ as follows: GenomeSpaceFilePath^Token -->
+        <param name="genomespace_browser" type="genomespacefile" label="Choose Target Directory" select_type="FOLDER" />
+        <param name="filename" type="text" label="Filename" help="Leave blank to generate automatically" />
     </inputs>
     <outputs>
-        <data format="html" name="output_log" />
+        <data format="auto" name="output_log" />
     </outputs>
     <help>
-This Tool allows you to export data to GenomeSpace. You must have logged in using your GenomeSpace OpenID. You can associate your OpenID credentials under the User Preferences panel.
+This Tool allows you to export data to GenomeSpace. Click the Browse button to select a file to import. The tool will automatically
```

Comment:
Thanks for catching that! Fixed in: https://github.com/galaxyproject/galaxy/pull/1814/commits/efc078cff1c6dd58462ec1a6f74bb38da83f0483